packages/material-ui/src/Step/Step.js

```diff
@@ -41,7 +41,7 @@ const Step = React.forwardRef(function Step(props, ref) {
     disabled = false,
     index,
     last,
-    orientation,
+    orientation = 'horizontal',
```

Comment:
Do we need the default value? Can we expect it to be always provided by the parent Stepper component? (it's the same question for Step Button)

Reply:
Just code documentation

Reply:
I'm in favor of not having this default value so that it doesn't hide a potential issue with the parent component.